src/app/assets/languages/en.json

```diff
@@ -214,7 +214,56 @@
           "index-storage-entity-helper": "Each entity that you specify must be unique.",
           "secured": "Security authorization",
           "secured-description": "Restrict access to the cache to protect data. {{brandname}} denies cache operations if users do not have one of the roles you select.",
-          "select-roles": "Select roles"
+          "select-roles": "Select roles",
+          "backups": "Backups",
+          "backups-description": "Configure {{brandname}} to replicate data in the cache to another cluster.",
+          "backups-site": "Backup locations",
+          "backups-site-tooltip": "Specify one or more remote sites to which {{brandname}} backs up data in the cache, for example LON or NYC. The remote site name that you enter must exist in the JGroups RELAY stack for the cluster configuration.",
```

Comment:
"Select one or more remote sites to which {{brandname}} backs up data in the cache"

The rest of the information should be gone, since it's not helpful and we are providing the site names with the REST API